tests/testapp/models.py

```diff
@@ -18,14 +27,35 @@ class PageWithColorField(Page):
 class PageWithCharColorField(Page):
     color = models.CharField(max_length=7)
 
+    content_panels = Page.content_panels + [
+        FieldPanel("color"),
+    ]
+
 
 class PageWithDefaultValue(Page):
     color = ColorField(default="#FF0000")
 
+    content_panels = Page.content_panels + [
+        NativeColorPanel("color"),
```

Comment:
I am just working on a similar kind of Widget implementation when using Wagtail 3 FieldPanel.

I don't know if this is relevant here or tested it out but it could be possible to use something like this. Having said that your changes will also work.

```suggestion
        FieldPanel("color", widget=ColorInputWidget),
```

Reply:
Yes it's possible, but I think we need to use `NativeColorPanel` because that would a programmer do based on their documentation.